lib/rails_response_dumper/runner.rb

```diff
@@ -12,7 +12,7 @@ def initialize(options)
     end
 
     def run_dumps
-      dumps_dir = Rails.root.join('dumps')
+      dumps_dir = ENV['DUMPS_DIR'] || Rails.root.join('dumps')
```

Comment:
I think this should be a proper CLI arg rather than an environment variable. The default of the CLI arg could be the original "dumps".

Reply:
Added in [Add cli arg to specify dumps directory](https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/71/commits/b168d13cb51201e44869cdc3415d2d1899b6bfff)